src/compiler/python_generator.cc

```diff
@@ -101,6 +101,11 @@ PrivateGenerator::PrivateGenerator(const GeneratorConfiguration& config,
 void PrivateGenerator::PrintAllComments(StringVector comments,
                                         grpc_generator::Printer* out) {
   if (comments.empty()) {
+    // It's important we print something here and not omit this line,
```

Comment:
In all cases of a service with no protodoc or just in the case of a service with no protodoc and no methods?